src/plone/restapi/serializer/__init__.py

```diff
@@ -93,7 +98,30 @@ def SerializeToJson(context):
 
     for schema in iterSchemata(context):
         for field_name, field in getFields(schema).items():
+            read_permissions = mergedTaggedValueDict(
+                schema, READ_PERMISSIONS_KEY)
+
+            if not check_permission(context, read_permissions.get(field_name)):
+                continue
             value = lookup_field_serializer(context, field)()
             result[json_compatible(field_name)] = value
 
     return result
+
+
+def check_permission(context, permission_name):
```

Comment:
Is this code dublicated from: https://github.com/plone/plone.restapi/blob/44ac5b4242349d99fe65ba6f89400cd955a3e8ce/src/plone/restapi/deserializer/dxcontent.py#L113 why not import it and reuse?